storage/postgres/postgres.go

```diff
@@ -325,7 +326,15 @@ func (p *Postgres) ReadAuthorizationModel(ctx context.Context, store string, mod
 		return nil, storage.ErrNotFound
 	}
 
+	// Update the schema version lazily
+	if version == typesystem.SchemaVersionUnspecified {
+		version = typesystem.SchemaVersion1_0
+		// Don't worry if we error, we'll update it lazily next time
+		_, _ = p.pool.Exec(ctx, "UPDATE authorization_model SET schema_version = $1 WHERE store = $2 AND authorization_model_id = $3", version, store, modelID)
```

Comment:
We should still log the error in case we screwed something up in the statement itself, because it's possible this could always error and we'd never know about it.

Reply:
Good point. Thanks. Updated.